src/lib/mappers/utils.js

```diff
@@ -1013,28 +1013,27 @@ module.exports = {
         mappedObject["Object.identificator"].push(id);
     },
 
-    mapIIIFManifest(input, mappedObject) {
+    mapIIIFManifest(input, mappedObject, utils) {
         if (input["object_number"]) {
+            const priref = input["@attributes"].priref;
+            const institution =  utils.getInstitutionNameFromPriref(priref);
             const objectNumber = input["object_number"][0];
-            const manifestURI = `https://api.collectie.gent/iiif/presentation/v2/manifest/${objectNumber}`;
+            const manifestURI = `https://api.collectie.gent/iiif/presentation/v2/manifest/${institution}:${objectNumber}`;
             const IIIFManifest = {
-                "subject_of": {
-                    "id": manifestURI,
-                    "type": "DigitalObject",
-                    "conforms_to": [{
-                        "id": "http://iiif.example.org/presentation/1/manifest.json",
-                        "type": "InformationObject"
-                    }],
-                    "format": "application/ld+json;profile=\"http://iiif.io/api/presentation/3/context.json\""
-                }
+                "id": manifestURI,
+                "type": "DigitalObject",
+                "conforms_to": [{
+                    "id": "http://iiif.example.org/presentation/1/manifest.json",
+                    "type": "InformatieObject"
+                }],
+                "format": 'application/ld+json;profile=\\"http://iiif.io/api/presentation/3/context.json\\"'
```

Comment:
Klopt dit wel?
```suggestion
                "format": 'application/ld+json;profile="http://iiif.io/api/presentation/3/context.json"'
```
of
```suggestion
                "format": "application/ld+json;profile=\"http://iiif.io/api/presentation/3/context.json\""
```

Reply:
was de bedoeling om hetzelfde te doen als Linked art: https://linked.art/model/digital/#iiif-manifests